ci-operator/step-registry/hypershift/mce/power/heterogeneous/OWNERS

```diff
@@ -0,0 +1,13 @@
+approvers:
```

Comment:
All the places please add @srivastav-abhishek too.

Reply:
Added!